bigtop-packages/src/charm/spark/layer-spark/lib/charms/layer/bigtop_spark.py

```diff
@@ -400,6 +400,10 @@ def start(self):
             else:
                 hookenv.log("Spark Worker did not start")
 
+        # Open port for Thrift Server
+        if hookenv.config()['spark_enable_thriftserver']:
```

Comment:
@kwmonroe I stuck this in here, but then realised that we don't cycle spark on a thrift server start, so I'm tempted to just stick it in with the start/stop logic for the thrift stuff. WDYT?